MobileSDK/sdk/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'android-library'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.2"
+
+    defaultConfig {
+        minSdkVersion 15
+        targetSdkVersion 23
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
```

Comment:
Still need to add that back in.  I could use your help with that, as I don't understand it very well.